### ISSUE TITLE
Loose Changelog

### ISSUE THREAD

**Issue by Sherlouk** [rnystrom/GitHawk#1372]
With the amount of TF builds that rocket out and (at times) the large de-sync between app store and TF builds I wonder if it'd be helpful to have a basic changelog?

There's been a number of issues recently where I think a lot of us have forgotten what's gone out and what hasn't!

Not saying we document *every* change, but at least a high level list of affected areas and prominent changes etc

Tagged as question to see if anyone else thinks it'd be helpful

cc @BasThomas @rnystrom

**Comment by BasThomas** [comment]
[Yes](https://github.com/rnystrom/GitHawk/issues/554)

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
If someone else wants to champion this I’m all for it, but adding this on is gonna be too much work for me.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Sherlouk** [comment]
1.10.0:
- Search and Browse Repositories
- GitHawk Signature on Comments
- iOS 11 Support

1.11.0:
- Edit and Delete Comments
- New Issue Creation
- Error "Toast" UI

1.12.0:
- Bookmark Repos/Issues/PRs to save for later
- Image Upload

1.13.0:
- Add & Remove Milestones from Issues/PRs

1.14.0:
- Assign people to Issues/PRs
- Double tap comments to react with 👍 
- Issue references appear in the timeline

1.15.0:
- Add reviewers to Issues/PRs
- Syntax Highlighting
- Basic rendering of checkboxes
- Tap images in HTML to open fullscreen
- Fixes to PR where files were not visible or could not be rendered

1.16.0:
- Accessibility improvements
- Attributed text for labels rendering
- Milestone progress bar on Issues/PRs
- Changes count added to PRs
- Inbox Zero is powered by Firebase

1.17.0:
- Checkboxes can be toggled by tapping on them
- View notifications for a repository
- Replaced SlackTextViewController dependency with our own version
- New "Card" based design tweaks
- Design tweaks to Issue/PR manage area
- Scroll performance tweaks in code viewer

**Comment by rnystrom** [comment]
Awesome! If you wanna PR it up and maintain it, all yours!

**Comment by Sherlouk** [comment]
Not gonna lie it's probably easier to just leave it as a comment, then any colloborator and can just update it super easily without having to open a PR etc? Removes the resistance?

**Comment by BasThomas** [comment]
You mean to keep the changelog in an/this issue instead of its own file?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Sherlouk** [comment]
Yup, happy to make it the root comment, delete comments, n lock it if it makes it easier? But still IMO easier to edit collaboratively

**Comment by BasThomas** [comment]
Yes, let’s try it out!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Guess we should close this?
